app/services/fixture_indexing_service.rb

```diff
@@ -62,10 +62,7 @@ def build_solr_document(id_prefix, oid, data_hash)
       oid_ssim: data_hash["oid"] || oid,
       identifierMfhd_ssim: data_hash["identifierMfhd"],
       identifierShelfMark_ssim: data_hash["identifierShelfMark"],
-      # I do not think the current box_ssim is how we want to continue to do deal with differences in field names
-      # However I do not think we currently have enough information to create the alternative (Max)
-      # Ladybird data_hash["box"] || Voyager data_hash["volumeEnumeration"] || ArchiveSpace data_hash["containerGrouping"]
-      box_ssim: data_hash["box"] || data_hash["volumeEnumeration"] || data_hash["containerGrouping"],
+      box_ssim: calculate_box_ssim(data_hash),
```

Comment:
FWIW - as a math major, I read the word "calculate" and expect the method to be doing some kind of arithmetic computation.  For this kind of thing, I might use names like "extract" "read" "parse".